support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
```

Comment:
Ok, this was a temporary hack which must be removed. I managed to override rpmlint errors by adding rpmlintrc file in the OBS package directory. Not sure how to deal with it in other build environments.